runtime/src/main/cpp/Worker.cpp

```diff
@@ -259,11 +259,14 @@ class State {
   }
 
   Worker* addWorkerUnlocked(bool errorReporting, KRef customName) {
-    Locker locker(&lock_);
-    Worker* worker = konanConstructInstance<Worker>(nextWorkerId(), errorReporting,
-        customName);
-    if (worker == nullptr) return nullptr;
-    workers_[worker->id()] = worker;
+    Worker* worker = nullptr;
```

Comment:
Why the function is called *****Unlocked while there is a lock used in it?

Reply:
Just a convention.